### DISCUSSION TITLE
IOI 2021 Training Camp Papers

### DISCUSSION THREAD

**Discussion by nhatsmrt** [enkerewpo/OI-Public-Library#16] [General]
Hi, I believe the IOI 2021 training camp papers are available, e.g [here](https://download.csdn.net/download/baidu_21084123/15874676?utm_medium=distribute.pc_relevant_download.none-task-download-2~default~BlogCommendFromBaidu~default-1.nonecase&dist_request_id=1328680.4305.16160966384052223&depth_1-utm_source=distribute.pc_relevant_download.none-task-download-2~default~BlogCommendFromBaidu~default-1.nonecase)

Could you add it? Thanks!

**Comment by enkerewpo** [comment]
Hi there, 
I haven't use csdn.net to download resources before, so it's kinda inconvenient. Because I don't have a paying method at csdn.net, while a vip subscription is required to download resources. 😥

**Comment by EndlessCheng** [comment]
[2021集训队论文初稿合集 页码待修.pdf](https://github.com/enkerewpo/OI-Public-Library/files/6289211/2021.pdf)

**Comment by nhatsmrt** [comment]
@enkerewpo no worries!
@EndlessCheng Thanks a lot!

**Comment by enkerewpo** [comment]
@EndlessCheng Amazing!  Feel free to make a pull request which can contribute to the public library 😀
